### ISSUE TITLE
[Bug]: TranslucentTB not opening

### ISSUE THREAD

**Issue by muiand** [TranslucentTB/TranslucentTB#1059]
### Please check for a duplicate first

- [X] I have checked the issue tracker if my bug has already been fixed or answered in another (possibly closed) issue.

### Steps to reproduce the issue

Download TranslucentTB and try to open - on windows 10

### Expected results

App opens

### Actual results

Several error message pop up (images attached)
![Capture](https://github.com/user-attachments/assets/049c6b4d-13d5-4d70-9165-17ac63cad4ae)
![image](https://github.com/user-attachments/assets/2c4eb69e-4e0d-4116-a152-52cefa58d1ed)
![image](https://github.com/user-attachments/assets/6aec775c-ad6d-40b5-a866-c697be16a522)
![image](https://github.com/user-attachments/assets/ce161679-ef22-4eb3-9eea-a64147370342)

### Windows version

Windows 10 2004-22H2 (builds 19041-19045)

### Other details

I already tried deleting and reinstalling several times, from both GitHub and Microsoft Store

### Relevant log output

```shell
C:\Program
Files\WindowsApps\Microsoft.VCLibs.140.00.UWPDesktop_14. 0.33728.0_x64_8wekyb3d8bbwe\MSVCP140_ATOMIC_WAIT.dll
is either not designed to run on Windows or it contains an error. Try installing the program again using the original installation media or contact your system administrator or the software vendor for support. Error status 0xc0000020.
```

**Comment by sylveon** [comment]
This looks like corruption (that error code means Bad Image). These are system dependencies, so I'm surprised you're not having issues with other store apps.

**Comment by muiand** [comment]
You're right, I have the issue with every microsoft store app (I don't use many store apps so I didn't notice), sorry. After trying sfc and dism fixes it still doesn't work though, I think I might give up. Thanks for the help though.
